out-adapter-persistence-jpa/src/main/java/gohigher/application/entity/ApplicationJpaEntity.java

```diff
@@ -104,6 +106,13 @@ public Application toDomain() {
 		return createApplication(processes, findCurrentProcess(processes));
 	}
 
+	public Application toDomain(List<ApplicationProcessJpaEntity> applicationProcessJpaEntities) {
+		List<Process> processes = applicationProcessJpaEntities.stream()
+			.map(ApplicationProcessJpaEntity::toDomain)
+			.toList();
+		return createApplication(processes, processes.get(currentProcessOrder));
```

Comment:
```suggestion
		return createApplication(processes, findCurrentProcess(processes));
```
process의 경우 order와 currentProcess의 Index번호간 차이가 발생할 수 있어 아래의 메서드로 변경해주시면 좋을 것 같아요!!
아래의 메서드를 사용해야 currentProcess를 찾는 로직의 변경이 수월할 것 같습니다

Reply:
변경했습니다!